logging/cloud-client/src/test/java/com/example/logging/LoggingIT.java

```diff
@@ -39,6 +39,9 @@
 @SuppressWarnings("checkstyle:abbreviationaswordinname")
 public class LoggingIT {
 
+  private final String QUICKSTART_LOG = "my-log";
```

Comment:
Should these go to $TEMP?

Reply:
This is the name of the log in StackDriver that it can later be referred by, not the location it's logging too.

Reply:
SGTM